src/extension/flutter/flutter_daemon.ts

```diff
@@ -65,6 +71,16 @@ export class FlutterDaemon extends StdIOService<UnknownNotification> implements
 		}
 	}
 
+	protected createNcProcess(port: string) {
+		vs.window.showInformationMessage(`About to create nc process with port: ${port}`);
```

Comment:
nit: this might read better as "Created nc process..." since by the time the user sees it, the process will almost certainly have been spawned.

Do we need to worry about `nc` not being available? (I guess users won't set the port number in config unless they're set up to use it, but if there might be situations where it's not there or not on `PATH` we might want to handle the error spawning the process and provide a message using `vs.window.showErrorMessage`).

Reply:
Oops not even sure I want this message. I think I'll delete it as it's more for debugging.

I _think_ `nc` should always be available for users, but `handleError` should log if there's a problem, right? Although would it be hard for users to capture this error message on startup? If users are having problems I can also have them set the log file locations in their settings.

Reply:
Oops, missed this. I think `handleError` might only logging to logs/console. I'm not sure exactly how it'll fail if it's not available. You could perhaps test this by changing `nc` to just some invalid command `notnc` and see how it behaves.

Reply:
I added an error message, though I'll probably change this to link to an error message defined internally in the future.